.travis.yml

```diff
@@ -86,11 +85,12 @@ matrix:
         TOOLCHAIN=osx-10-13-cxx14
         PROJECT_DIR=examples/Assimp
 
-    - os: osx
-      osx_image: xcode9.4
-      env: >
-        TOOLCHAIN=ios-nocodesign-11-4-dep-9-3
-        PROJECT_DIR=examples/Assimp
+    #FIXME: https://travis-ci.com/rbsheth/hunter/jobs/209825923
```

Comment:
iOS is working fine, please revert:
- https://travis-ci.org/ingenue/hunter/builds/552984728

Reply:
Fixed